src/mixins/amalgamation-mixin.ts

```diff
@@ -22,36 +24,39 @@ export default class AmalgamationMixin extends Vue {
     this.limitedRestoration,
     this.futureEffectiveFiling,
     this.foreign,
-    this.foreignUlc,
+    this.foreignUnlimited,
     this.cccMismatch,
-    this.ulcMismatch
+    this.foreignUnlimited2,
+    this.xproUlcCcc,
+    // this.needBcCompany,
+    this.foreignHorizontal
   ]
 
   /** If we don't have an address, assume business is not affiliated (except for staff). */
   notAffiliated (business: AmalgamatingBusinessIF): AmlStatuses {
-    if (business.type === AmlTypes.LEAR && !business.address && !this.isRoleStaff) {
+    if (!this.isRoleStaff && business.type === AmlTypes.LEAR && !business.address) {
```

Comment:
By moving this check first, this if statement will resolve faster for staff users.